algebird-test/src/test/scala/com/twitter/algebird/SpaceSaverTest.scala

```diff
@@ -59,5 +61,26 @@ class SpaceSaverTest extends WordSpec with Matchers {
     "be consistent in its constructors" in {
       assert(SpaceSaver(10, "ha") ++ SpaceSaver(10, "ha") ++ SpaceSaver(10, "ha") === SpaceSaver(10, "ha", 3))
     }
+
+    "Correctly serialize" in {
+      //SSOne
+      verifySerialization(SpaceSaver(40, "string that is not empty"))
```

Comment:
would you mind converting this to a scalacheck test? Something like

```scala
property("SpaceSaver serializes properly") { ss: SpaceSaver[String] =>
  val spaceSaverAsArrayByte = SpaceSaver.toBytes(ss, stringToArrayByte)
  SpaceSaver.fromBytes(spaceSaverAsArrayByte, arrayByteToString).get == ss
}
```

The best way to make this work is to get a scalacheck `Arbitrary` instance into this file:

https://github.com/twitter/algebird/blob/develop/algebird-test/src/main/scala/com/twitter/algebird/scalacheck/Arbitrary.scala

and a `Gen` instance into this file:

https://github.com/twitter/algebird/blob/develop/algebird-test/src/main/scala/com/twitter/algebird/scalacheck/Gen.scala

I know it's a bigger ask, but serialization is a big deal and I think we absolutely need the scalacheck laws in place here.